app/controllers/users_controller.rb

```diff
@@ -0,0 +1,46 @@
+class UsersController < ApplicationController
+  def logged
```

Comment:
as previously discussed, this method is not consistent. If it returns a bool it should end with '?'

if the purpose of the method is to find out if the user is logged or not it should return only true/false

this one also renders and sets the status code